playbooks/setup.yml

```diff
@@ -5,6 +5,23 @@
 # Use: `ansible-playbook playbooks/setup.yml -e "ansible_user=ubuntu" --limit <server_tag>` if
 # provisioning an EC2 instance for the first time (this only needs to be run once).
 
+# Ensure python is set up and accessible on the server before starting.
+- name: python check
+  hosts: ofn_servers
+  gather_facts: no
+  remote_user: root
+  tasks:
+    - name: Install python
+      become: yes
+      raw: |
+        test -e /usr/bin/python || (
```

Comment:
Should we also test for `/usr/bin/python3` ? we could skip the install if already there ?

Reply:
Currently, we depend on that symlink:

https://github.com/openfoodfoundation/ofn-install/blob/e1c518166463c1199ceb6c53b544783fd0bde0ac/inventory/group_vars/all.yml#L410-L411

But we could experiment with removing that config. I would expect Ansible to be clever enough to find any python on the machine.